nautobot/vpn/api/serializers.py

```diff
@@ -97,12 +97,33 @@ class Meta:
 class VPNTunnelSerializer(TaggedModelSerializerMixin, NautobotModelSerializer):  # pylint: disable=too-many-ancestors
     """VPNTunnel Serializer."""
 
+    vpn_profile = SerializerMethodField(read_only=True)
```

Comment:
Let's hold off on this until we have agreement on a design. One thing to add in a single app, another thing to create another standard in core.

Reply:
Sure, converting PR to draft for future consideration.